pkg/operator/edgefs/cluster/configmap.go

```diff
@@ -48,17 +48,51 @@ func (c *cluster) createClusterConfigMap(deploymentConfig edgefsv1beta1.ClusterD
 
 	serverIfName := defaultServerIfName
 	brokerIfName := defaultBrokerIfName
-	if isHostNetworkDefined(c.Spec.Network) {
-
-		if len(c.Spec.Network.ServerIfName) > 0 && len(c.Spec.Network.BrokerIfName) > 0 {
-			serverIfName = c.Spec.Network.ServerIfName
-			brokerIfName = c.Spec.Network.BrokerIfName
-		} else if len(c.Spec.Network.ServerIfName) > 0 {
-			serverIfName = c.Spec.Network.ServerIfName
-			brokerIfName = c.Spec.Network.ServerIfName
-		} else if len(c.Spec.Network.BrokerIfName) > 0 {
-			serverIfName = c.Spec.Network.BrokerIfName
-			brokerIfName = c.Spec.Network.BrokerIfName
+
+	serverSelector, serverDefined := c.Spec.Network.Selectors["server"]
+	brokerSelector, brokerDefined := c.Spec.Network.Selectors["broker"]
+
+	if c.Spec.Network.IsHost() {
+		if serverDefined && brokerDefined {
+			serverIfName = serverSelector
+			brokerIfName = brokerSelector
+		} else if serverDefined {
+			serverIfName = serverSelector
+			brokerIfName = serverSelector
+		} else if brokerDefined {
+			serverIfName = brokerSelector
+			brokerIfName = brokerSelector
+		}
+	} else if c.Spec.Network.IsMultus() {
+		if serverDefined && brokerDefined {
+			var err error
+			serverIfName, err = k8sutil.GetMultusIfName(serverSelector)
+
```

Comment:
nit: Would you mind removing blank lines before the `if err != nil` statements? It's uncommon to have that whitespace.

Reply:
Ok